superset-frontend/src/datasource/ChangeDatasourceModal.tsx

```diff
@@ -224,6 +221,7 @@ const ChangeDatasourceModal: FunctionComponent<ChangeDatasourceModalProps> = ({
       onHide={onHide}
       responsive
       title={t('Change Dataset')}
+      height="350px"
```

Comment:
isn't better to send height without 'px'?

Reply:
I think it is preferable to avoid specifying px just in case another unit needs to be used, see a percentage, for instance.